react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,13 @@
+.fa-times {
+  color: red;
+  margin-left: 10px;
+}
+
+.fa-check {
+  color: green;
+  margin-left: 10px;
+}
```

Comment:
You don't want to apply CSS rules to existing classes from CSS libraries. This can cause issues in other parts of your app and will cause confusion with other developers working on the app. The expected behavior of the class will vary from the official documentation from the CSS library. Here it would be best to make your own classes and apply them to which ever elements need them. For example:

```CSS
.icon {
  margin-left: 10px;
}

.text-green {
  color: green;
}

.text-red {
  color: red;
}
```